### PR TITLE
Prep amazon.aws 6.5.3

### PR DESCRIPTION
**Pull request by alinabuzachis**
##### SUMMARY

Prep amazon.aws 6.5.3
Release date is March 5th.

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

several

### CODE REVIEW

File: changelogs/changelog.yaml [hunk 1]
```diff
@@ -2396,3 +2396,19 @@ releases:
...
     release_date: '2024-02-06'
+  6.5.3:
+    changes:
+      bugfixes:
+      - cloudwatchevent_rule - Fix to avoid adding quotes to JSON input for provided
+        input_template (https://github.com/ansible-collections/amazon.aws/pull/1883).
+      - lookup/secretsmanager_secret - fix the issue when the nested secret is missing
+        and on_missing is set to warn, the lookup was raising an error instead of
+        a warning message (https://github.com/ansible-collections/amazon.aws/issues/1781).
+      release_summary: This release bugfixes for the``cloudwatchevent_rule`` module
```
Comment (GomathiselviS) [inline changelogs/changelog.yaml:2407]:
```suggestion
      release_summary: This release includes bugfixes for the``cloudwatchevent_rule`` module
```

**Review by GomathiselviS — APPROVED** [review]

File: changelogs/changelog.yaml [hunk 2]
```diff
@@ -2396,3 +2396,19 @@ releases:
...
     release_date: '2024-02-06'
+  6.5.3:
+    changes:
+      bugfixes:
+      - cloudwatchevent_rule - Fix to avoid adding quotes to JSON input for provided
+        input_template (https://github.com/ansible-collections/amazon.aws/pull/1883).
+      - lookup/secretsmanager_secret - fix the issue when the nested secret is missing
+        and on_missing is set to warn, the lookup was raising an error instead of
+        a warning message (https://github.com/ansible-collections/amazon.aws/issues/1781).
+      release_summary: This release includes bugfixes for the``cloudwatchevent_rule`` module
```
Comment (tremble) [inline changelogs/changelog.yaml:2407]:
```suggestion
      release_summary: This release includes bugfixes for the ``cloudwatchevent_rule`` module
```

File: CHANGELOG.rst [hunk 3]
```diff
@@ -5,6 +5,20 @@ amazon.aws Release Notes
 .. contents:: Topics
 
 
+v6.5.3
+======
+
+Release Summary
+---------------
+
+This release includes bugfixes for the``cloudwatchevent_rule`` module and ``secretsmanager_secret`` lookup plugin.
```
Comment (tremble) [inline CHANGELOG.rst:14]:
```suggestion
This release includes bugfixes for the ``cloudwatchevent_rule`` module and ``secretsmanager_secret`` lookup plugin.
```

**Review by tremble — APPROVED** [review]

**Comment by alinabuzachis** [comment]
recheck

File: changelogs/changelog.yaml [hunk 4]
```diff
@@ -2396,3 +2396,19 @@ releases:
...
+        and ``secretsmanager_secret`` lookup plugin.
+    fragments:
+    - 1883-cloudwatchevent_rule-fix-json-input-handling-for-input_template.yml
+    - 20240212-lookup-secretsmanager_secret-fix-issue-when-nested-is-missing-and-on_missing-is-set-to-warn.yaml
+    - release_summary.yml
+    - remove_rendundant_workflow.yml
+    release_date: '2024-03-05'
```
Comment (tremble) [inline changelogs/changelog.yaml:2414]:
```suggestion
    release_date: '2024-03-06'
```

**Pull request merged**